vagrant_support/my.cnf

```diff
@@ -0,0 +1,46 @@
+[mysqld]
+datadir=/var/lib/mysql
+socket=/var/lib/mysql/mysql.sock
+user=mysql
+# Disabling symbolic-links is recommended to prevent assorted security risks
+symbolic-links=0
+
+default_storage_engine = InnoDB
+key_buffer_size        = 32M
+myisam_recover         = FORCE,BACKUP
+max_allowed_packet     = 1G
+max_connect_errors     = 1000000
+innodb                 = FORCE
+datadir                = /var/lib/mysql
+character-set-server   = utf8
+collation-server       = utf8_general_ci
+
+tmp_table_size         = 32M
+max_heap_table_size    = 32M
+query_cache_type       = 0
+query_cache_size       = 0
+max_connections        = 500
+thread_cache_size      = 50
+#open_files_limit       = 65535
+table_definition_cache = 1024
+table_open_cache       = 2048
+
+innodb_flush_method            = O_DIRECT
+innodb_log_files_in_group      = 2
+innodb_log_file_size           = 256M
+innodb_flush_log_at_trx_commit = 2
+innodb_file_per_table          = 1
+innodb_buffer_pool_size        = 1G
```

Comment:
Is 1GB the appropriate buffer pool size for the average / 95th percentile Vagrant BMO installations? RAM is precious again these days thanks to lots of parallel VMs and if there's massive savings to be had here vs. everyday normal use cases, I'm in favor of reducing it if no other harm comes to us.